state-sync/diem-data-client/src/diemnet/logging.rs

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::Error;
+use diem_logger::Schema;
+use serde::Serialize;
+
+#[derive(Schema)]
+pub struct LogSchema<'a> {
+    name: LogEntry,
+    error: Option<&'a Error>,
+    event: Option<LogEvent>,
+    message: Option<&'a str>,
+}
```

Comment:
Is there any existing data client ID field that we can use to identify a log entry?

Reply:
Is this to identify the machine or the individual client on the server(is there supposed to be more than one?)

Reply:
@khiemngo, there is no data client ID field in this context (I don't think we really need it). Especially as users of the diem data client will also do their own logging, and the data client relies on the network implementation to handle request/response mappings.